web/styles/input_pill.css

```diff
@@ -120,7 +120,7 @@
 }
 
 #compose-direct-recipient.pill-container {
-    padding: 0 2px;
```

Comment:
Removing this padding should not be a problem since pills already have margin to space themselves